math/pari/Portfile

```diff
@@ -4,7 +4,7 @@ PortSystem          1.0
 PortGroup           muniversal 1.0
 
 name                pari
-version             2.11.1
+version             2.11.4
 revision            1
```

Comment:
The `revision` needs to be set back to 0 when increasing the `version`.

Reply:
Good catch, should be fixed now.